prisma/schema.prisma

```diff
@@ -13,88 +13,89 @@ model Snl {
-  photo               Upload?              @relation(fields: [photoId], references: [id])
-  referrer            User?                @relation("referrals", fields: [referrerId], references: [id])
-  referrees           User[]               @relation("referrals")
-  Account             Account[]
-  Session             Session[]
-  itemForwards        ItemForward[]
-  hideBookmarks       Boolean              @default(false)
-  followers           UserSubscription[]   @relation("follower")
-  followees           UserSubscription[]   @relation("followee")
-  hideWelcomeBanner   Boolean              @default(false)
-  diagnostics         Boolean              @default(false)
-  hideIsContributor   Boolean              @default(false)
-  muters              Mute[]               @relation("muter")
-  muteds              Mute[]               @relation("muted")
+  id                    Int                  @id @default(autoincrement())
+  createdAt             DateTime             @default(now()) @map("created_at")
+  updatedAt             DateTime             @default(now()) @updatedAt @map("updated_at")
+  name                  String?              @unique(map: "users.name_unique") @db.Citext
+  email                 String?              @unique(map: "users.email_unique")
+  emailVerified         DateTime?            @map("email_verified")
+  image                 String?
+  msats                 BigInt               @default(0)
+  freeComments          Int                  @default(5)
+  freePosts             Int                  @default(2)
+  checkedNotesAt        DateTime?
+  pubkey                String?              @unique(map: "users.pubkey_unique")
+  tipDefault            Int                  @default(100)
+  bioId                 Int?
+  inviteId              String?
+  tipPopover            Boolean              @default(false)
+  upvotePopover         Boolean              @default(false)
+  trust                 Float                @default(0)
+  lastSeenAt            DateTime?
+  stackedMsats          BigInt               @default(0)
+  noteAllDescendants    Boolean              @default(true)
+  noteDeposits          Boolean              @default(true)
+  noteEarning           Boolean              @default(true)
+  noteInvites           Boolean              @default(true)
+  noteItemSats          Boolean              @default(true)
+  noteMentions          Boolean              @default(true)
+  noteForwardedSats     Boolean              @default(true)
+  lastCheckedJobs       DateTime?
+  noteJobIndicator      Boolean              @default(true)
+  photoId               Int?
+  upvoteTrust           Float                @default(0)
+  hideInvoiceDesc       Boolean              @default(false)
+  wildWestMode          Boolean              @default(false)
+  greeterMode           Boolean              @default(false)
+  fiatCurrency          String               @default("USD")
+  withdrawMaxFeeDefault Int                  @default(10)
```

Comment:
This was the only line added (expanding the whole column)

Reply:
If there's a default here at the DB level, do we need to do all the `|| 21` in the code? Can't we assume there's always a value in `me.withdrawMaxFeeDefault`?

Reply:
Yes, what @SatsAllDay is saying is true if `me` is always queried with `withdrawMaxFeeDefault`

Reply:
Also agree with @SatsAllDay but just want to mention that additionally to the default, the column is not nullable. If it would be nullable this would be a slightly different story.

So I also think the `|| 21` are not necessary

Reply:
Removed, thanks all!